stdlib/tkinter/__init__.pyi

```diff
@@ -273,10 +273,13 @@ class Variable:
     def trace_add(self, mode: _TraceMode, callback: Callable[[str, str, str], object]) -> str: ...
     def trace_remove(self, mode: _TraceMode, cbname: str) -> None: ...
     def trace_info(self) -> list[tuple[tuple[_TraceMode, ...], str]]: ...
-    def trace_variable(self, mode, callback): ...  # deprecated
-    def trace_vdelete(self, mode, cbname) -> None: ...  # deprecated
-    def trace_vinfo(self): ...  # deprecated
-    trace = trace_variable  # deprecated
+    @deprecated("use trace_add() instead of trace_variable()")
```

Comment:
Is this scheduled for removal in a specific version? If so, let's add "will be removed in Python 3.14" (or whatever other version)

Reply:
There is no associated Python version. [Documentation for the Tcl commands](https://www.tcl.tk/man/tcl/TclCmd/trace.html) that these methods wrap are being deprecated doesn't specify a version either, it only says:

> These subcommands are deprecated and will likely be removed in a future version of Tcl.

Reply:
I see, makes sense.